rclcpp_components/src/component_container.cpp

```diff
@@ -22,8 +22,15 @@ int main(int argc, char * argv[])
 {
   /// Component container with a single-threaded executor.
   rclcpp::init(argc, argv);
+
+  rclcpp::NodeOptions options;
+  options.start_parameter_services(false);
+  options.start_parameter_event_publisher(false);
```

Comment:
although this makes sense, I would adjust these default options with constructor `ComponentManager::ComponentManager` since these options seem reasonable to be disabled and reduce redundant code.

Reply:
That was my original change, however while searching for owners of this code, I found a pull request to make `ComponentManager::ComponentManager` public, so that other packages it can be reused. If any derived class wants to introduce parameters it would be impossible to start using parameter services or the parameter event publisher unless this code is changed again.

References:
- https://github.com/ros2/rclcpp/issues/1010
- https://github.com/ros-controls/roadmap/pull/7

Reply:
Never mind my reply above, shortly after I typed it, I realized what you meant with 'default options'. Now it's in the default argument of the constructor.